apps/api_key_mgmt/lib/api_key_mgmt/handler.ex

```diff
@@ -144,12 +144,32 @@ defmodule ApiKeyMgmt.Handler do
            |> Auth.Context.put_operation("RevokeApiKey", party_id, api_key_id)
            |> Auth.Context.add_operation_entity(api_key)
            |> Auth.authorize(rpc_context: ctx.rpc) do
+      # TODO: Since we cant make this run as a transaction
+      # without modifying the current Authority service API
+      # be aware, that updating the database can fail, which in turn
+      # would result in a descrepancy between the state shown (active),
+      # and the ability to authenticate with such key (none).
+      # Temporaty fix: manually fix the database with an SQL query.
```

Comment:
```suggestion
      # Temporary fix: manually fix the database with an SQL query.
```

Ну а в целом мне кажется, что надо б тогда задачу расписать про то, как сделать правильно.